xwiki-platform-core/xwiki-platform-wiki/xwiki-platform-wiki-ui/xwiki-platform-wiki-ui-wiki/src/main/resources/WikiManager/Translations.xml

```diff
@@ -105,8 +105,8 @@ platform.wiki.currentwiki.joinRequestAcceptedNotification=Your request to join t
 platform.wiki.currentwiki.joinRequestRejectedNotification=Your request to join the wiki ''{0}'' has been rejected.
 platform.wiki.currentwiki.candidateHandledSaveComment=Handled candidate {0} ({1})
 platform.wiki.currentwiki.candidateHandledComment=Comment:
-platform.wiki.currentwiki.candidateAcceptedSuccess=Request accepted successfully.
-platform.wiki.currentwiki.candidateRejectedSuccess=Request rejected successfully.
+platform.wiki.currentwiki.candidateAcceptedSuccess=Request accepted,
+platform.wiki.currentwiki.candidateRejectedSuccess=Request rejected
```

Comment:
Same (missing completely in this case, should be added).

Reply:
Done.